src/chigame/games/models.py

```diff
@@ -34,13 +36,39 @@ class Game(models.Model):
     min_playtime = models.PositiveIntegerField(null=True, blank=True)
     max_playtime = models.PositiveIntegerField(null=True, blank=True)
 
-    complexity = models.PositiveSmallIntegerField(null=True, blank=True)  # 1-5, 1 being the easiest
-    category = models.ManyToManyField("Category", related_name="games", blank=True)
+    complexity = models.DecimalField(
+        max_digits=3, decimal_places=2, null=True, validators=[MinValueValidator(1), MaxValueValidator(5)]
+    )
+
+    categories = models.ManyToManyField("Category", related_name="games", blank=True)
     mechanics = models.ManyToManyField("Mechanic", related_name="games", blank=True)
 
     # ================ OTHER ================
     BGG_id = models.PositiveIntegerField(null=True, blank=True)  # BoardGameGeek ID
 
+    # ================ VALIDATON ================
+    def clean(self):
+        # Ensures min_players is not greater than max_players
+        if self.min_players and self.max_players and self.min_players > self.max_players:
```

Comment:
In the future as we potentially add more validation, consider turning each of these checks into helper functions to clean up this section of code.
As of now it's fine enough.